pkg/trace/obfuscate/obfuscate.go

```diff
@@ -44,6 +45,26 @@ type Config struct {
 
 	// Redis enables obfuscatiion of the "memcached.command" tag for spans of type "memcached".
 	Memcached bool
+
+	// literalEscapes reports whether we should treat escape characters literally or as escape characters.
+	// A non-zero value means 'yes'. Different SQL engines behave in different ways and the tokenizer needs
+	// to be generic.
+	// Not safe for concurrent use.
+	literalEscapes int32
```

Comment:
I just realised that this is the global obfuscator config applying to all obfuscators (SQL, memcached, elasticsearch, etc) so maybe we should be more specific here?

```suggestion
	sqlLiteralEscapes int32
```

What do you think?